src/app/components/feed/feed.component.scss

```diff
@@ -0,0 +1,18 @@
+@use '../../../colors';
```

Comment:
If you would want to, you should be able to use the following:
```
import '../../../colors';
color: $basicTextColor
```

that way, you don't need to always use `colors.$someColor` 

It's totally fine like this, just wanted to point out a different way :)

Reply:
Thank you, that's much better :)